lms/djangoapps/course_goals/models.py

```diff
@@ -46,3 +46,20 @@ def __str__(self):
             goal=self.days_per_week,
             course=self.course_key,
         )
+
+
+class UserActivity(models.Model):
+    """
+    Tracks the date a user performs an activity in a course for goal purposes.
```

Comment:
per the above comment, i would either rename this model to something that sounds more specific to the goals feature, and explain in the comment that the only types of activities that would be stored in this table are the types of activities that we count as going towards your goal

or if we don't want to do that, then

i would add back the activity_type field and move the table to a more generic location than the goals models

Reply:
Ah I see what you mean. Yeah, that's a good idea. I'll add into the docstring to mention that to start we will be tracking page views and in the future, it can grow to track other types of activities as well.